tests/elastio_snap.py

```diff
@@ -45,6 +49,18 @@
 int elastio_snap_get_free_minor(void);
 """)
 
+
+class Flags:
+    COW_REDIRECTED = 0
```

Comment:
Could you please clarify why we need this `COW_REDIRECTED` value? If I got this correctly, this is just an inverted value of `COW_ON_BDEV`. If this is true, then we need to change this logic to check `COW_ON_BDEV` bit rather than its inverted value (zero). We might have other flags in the future and it may break the logic.